risc0/zkp/Cargo.toml

```diff
@@ -58,7 +58,7 @@ features = ["prove", "std"]
 circuit_debug = []
 cuda = ["dep:cust", "prove", "risc0-sys/cuda"]
 default = []
-metal = []
+metal = ["prove"]
```

Comment:
```suggestion
metal = ["prove", "risc0-sys/metal"]
```

Reply:
I haven't originally included `risc0-sys/metal` as it is an empty feature.